packages/config/config/devices/0x010f/templates/fgwpx-1x1.json

```
@@ -0,0 +1,489 @@
+		},
+		"24": {
+			"label": "Switch On",
+			"description": "Command frame sent to devices in 2nd group",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "Last State/ON",
+					"value": 255
+				}
+			]
+		},
+		"25": {
+			"label": "Switch Off",
+			"description": "Command frame sent to devices in 2nd group",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "Last State/ON",
+					"value": 255
+				}
+			]
+		},
+		"26": {
+			"label": "Threshold Up",
+			"description": "Command frame sent to the devices in 3rd group",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "Last State/ON",
+					"value": 255
+				}
+			]
+		},
+		"27": {
+			"label": "Threshold Down",
+			"description": "Command frame sent to the devices in 3rd group",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "Last State/ON",
+					"value": 255
+				}
+			]
+		},
+		"30[0x01]": {
+			"$import": "fibarogroup_templates.json#enable_disable_bit_default_true",
+			"label": "React to General Alarm",
+			"description": "Network alarms to which the Wall Plug will respond"
+		},
+		"30[0x02]": {
+			"$import": "fibarogroup_templates.json#enable_disable_bit_default_true",
+			"label": "React to Smoke Alarm",
+			"description": "Network alarms to which the Wall Plug will respond"
+		},
+		"30[0x04]": {
+			"$import": "fibarogroup_templates.json#enable_disable_bit_default_true",
+			"label": "React to Carbon Monoxide Alarm",
+			"description": "Network alarms to which the Wall Plug will respond"
+		},
+		"30[0x08]": {
+			"$import": "fibarogroup_templates.json#enable_disable_bit_default_true",
+			"label": "React to Carbon Dioxide Alarm",
+			"description": "Network alarms to which the Wall Plug will respond"
+		},
+		"30[0x0f]": {
+			"$import": "fibarogroup_templates.json#enable_disable_bit_default_true",
+			"label": "React to High Temperature Alarm",
+			"description": "Network alarms to which the Wall Plug will respond"
+		},
+		"30[0x10]": {
+			"$import": "fibarogroup_templates.json#enable_disable_bit_default_true",
+			"label": "React to Flood Alarm",
+			"description": "Network alarms to which the Wall Plug will respond"
+		},
```

```suggestion
		"30[0x10]": {
			"$import": "fibarogroup_templates.json#enable_disable_bit_default_true",
			"label": "React to High Temperature Alarm",
			"description": "Network alarms to which the Wall Plug will respond"
		},
		"30[0x20]": {
			"$import": "fibarogroup_templates.json#enable_disable_bit_default_true",
			"label": "React to Flood Alarm",
			"description": "Network alarms to which the Wall Plug will respond"
		},
```
I guess this is what you meant.